.github/workflows/lwcE2E.yml

```
@@ -0,0 +1,139 @@
+name: LWC End to End Tests
+
+on:
+  workflow_dispatch:
+    inputs:
+      automationBranch:
+        description: 'Set the branch to use for automation tests'
+        required: false
+        default: 'develop'
+        type: string
+      lwcLsp:
+        description: 'LWC LSP'
+        required: false
+        default: true
+        type: boolean
+      debugLwcTests:
+        description: 'Debug LWC Tests'
+        required: false
+        default: true
+        type: boolean
+      runLwcTests:
+        description: 'Run LWC Tests'
+        required: false
+        default: true
+        type: boolean
+      vscodeVersion:
+        description: 'VSCode Version'
+        required: false
+        default: '1.85.2'
+        type: string
+      runId:
+        description: 'Run ID of the workflow run that created the vsixes'
+        required: true
+        type: string
+
+  workflow_call:
+    inputs:
+      automationBranch:
+        description: 'Set the branch to use for automation tests'
+        required: false
+        default: 'develop'
+        type: string
+      lwcLsp:
+        description: 'Lwc LSP'
```

It's uppercase in the workflow_dispatch!

```suggestion
        description: 'LWC LSP'
```